### ISSUE TITLE
about the progress of this mod

### ISSUE THREAD

**Issue by pveenthusiast** [catornot/bp-ort#1]
hey there.. just stopping by after I heard about ur work on northstar and wanting to know how far have ya gotten.. I think it's a fantastic addition to the available mods on that client.. we also have "grunt mode" by ender9217 and "attrition extended" by voyagedb... the only thing those mods lack is.. pilots that move like pilots, ergo, npc's with advanced movement.

and by thinking about that, I know that programming them must be hard as hell, pretty much what activision had to deal with their bots on black ops 3 and advanced warfare. with that being said, in advance thanks for making it possible. also, remember that northstar added that "switching teams" feature to its client so.. mods that aint have it wont work properly.. i learned it today when I tried to play an attrition match with those mods I mentioned before.. one of them stopped getting updates so.. busted!

tldr: thanks for making this mod and remember to implement the "switching teams" feature in it.

**Comment by catornot** [comment]
are you referring to https://github.com/R2Northstar/NorthstarMods/pull/872 ?
since if that's the case idk how it be relevant to this plugin unless you also want a ui element to switch the team of bots.
( in any case bp-ort has commands for switching teams like in [admin abuse](https://thunderstore.io/c/northstar/p/x3Karma/Karma_Mod_Abuse/). with the switch command and grant__admin to give admin perms to a player via the server console or setting the [admin abuse](https://thunderstore.io/c/northstar/p/x3Karma/Karma_Mod_Abuse/)'s admin cvar )

**Comment by pveenthusiast** [comment]
what i could tell is that every outdated mod that i had stopped working at the moment that update was implemented on the northstar client.. there's another mod i have from ender9217, "grunt mode", and he mentioned on his changelog that made an update in regards of that "switing teams" thing, so i think that feature now is necessary so the mods can work properly... 
![Sin título](https://github.com/user-attachments/assets/f81a7d12-351b-4381-b2b5-3d32695e567a)

**Comment by catornot** [comment]
So this mod still work? 
If it does then I won't do anything since the switch teams button that was introduced in 1.29.0 does not affect this mod

**Comment by pveenthusiast** [comment]
yeah, the "grunt mod" from ender works.. however.. and this in regards of ur mod as such.. i tried loading it today and i got this message.. any idea why? i even copied the dll file in the plugins folder and everything.. 
![Sin título](https://github.com/user-attachments/assets/1e38c615-3538-4899-8e72-fcd802b5a4b3)

**Comment by pveenthusiast** [comment]
nevermind.. i loaded your mod again and this time it worked.. very nice, btw.. the command #6 is the best one.. those bots are a bunch of tryhards.. i lost 5-13 against them last time lol... my only complain would be that they dont use advanced movement, like wallrunning and jumping.. and secondly.. how do i set their names? i thought u had a list of names for them which applied automatically.

**Comment by catornot** [comment]
A list of names does exist but it's hard coded. You can set their names when spawning them. It's the first argument to the command. As for the advanced movement, that would be very complicated since the default navigation meshes (from the game) don't support wallrunning, only apex has that. I could try to get them to slidehop tho.

**Comment by pveenthusiast** [comment]
yeah.. i thought something like that was the reason.. i mean, besides lastimosa.. is there any other npc on the campaign that uses advanced movement? since u modders take most of the movement code lines from the npc's of the campaign, right? but well.. now i can play the rest of the multiplayer maps, thanks.

**Comment by catornot** [comment]
the movement is fully custom, but even in game none of the npcs use advanced movement, it's all prerecorded scripted animation.
